app/resources/api/v2/specific_tube_creation_resource.rb

```diff
@@ -0,0 +1,99 @@
+    #
+    # Provides a JSON:API representation of {SpecificTubeCreation}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
+    # of the JSON:API standard.
+    class SpecificTubeCreationResource < BaseResource
+      ###
+      # Attributes
+      ###
+
+      # @!attribute [w] child_purpose_uuids
+      #   @param value [Array] Array of UUIDs for child purposes to use in the creation of tubes.
+      #   @return [Void]
+      attribute :child_purpose_uuids
+
+      def child_purpose_uuids=(value)
+        @model.set_child_purposes(value)
+      end
+
+      # @!attribute [w] parent_uuid
+      #   This is declared for convenience where the parent is not available to set as a relationship.
+      #   Setting this attribute alongside the `parent` relationship will prefer the relationship value.
+      #   @param value [String] The UUID of a single parent plate that will be the parent for all tubes created.
+      #   @return [Void]
+      #   @see #parent
+      attribute :parent_uuid
+
+      def parent_uuid=(value)
+        @model.parent = Plate.with_uuid(value).first
+      end
+
+      # @!attribute [w] tube_attributes
+      #   @param value [Array<Hash>] Hashes defining the attributes to apply to each tube being created.
+      #     This might be used to set custom attributes on the tubes, such as name.
+      #   @example Setting the name of the tubes being created.
+      #     [{ name: 'Tube one' }, { name: 'Tube two' }]
+      #   @return [Void]
+      attribute :tube_attributes
+
+      # @!attribute [w] user_uuid
+      #   This is declared for convenience where the user is not available to set as a relationship.
+      #   Setting this attribute alongside the `user` relationship will prefer the relationship value.
+      #   @param value [String] The UUID of the user who initiated the creation of tubes.
+      #   @return [Void]
+      #   @see #user
+      attribute :user_uuid
+
+      def user_uuid=(value)
+        @model.user = User.with_uuid(value).first
+      end
+
+      # @!attribute [r] uuid
+      #   @return [String] The UUID of the state change.
+      attribute :uuid, readonly: true
+
+      ###
+      # Relationships
+      ###
+
+      # @!attribute [rw] user
+      #   Setting this relationship alongside the `user_uuid` attribute will override the attribute value.
+      #   @return [UserResource] The user who initiated the creation of tubes.
+      #   @note This relationship is required.
+      has_one :user
+
+      # @!attribute [rw] parents
+      #   Setting this relationship alongside the `parent_uuid` attribute will override the attribute value.
+      #   @return [Array<AssetResource>] An array of the parents of the tubes being created.
+      #   @note This relationship is required.
+      has_many :parents, class_name: 'Asset'
```

Comment:
I think this ought to be Labware rather than Asset as we're trying to deprecate that.

Reply:
Can discuss as had the same issue with my attempt at the tube rack tubes and not sure what to do there.